datasets/random.py

```diff
@@ -10,16 +10,15 @@ class Dataset(BaseDataset):
     # List of parameters to generate the datasets. The benchmark will consider
     # the cross product for each key in the dictionary.
     parameters = {
-        'n_samples, n_features': [
+        "n_samples, n_features": [
             (1000, 2000),
-            # (200, 100),
         ],
-        'ill_conditioned': [False],
+        "ill_conditioned": [False, True],
```

Comment:
I'd call it something even more frightening than "ill_conditioned", like "adversarial", because not only is `X` bad, but we take an adversarial `y` too. WDYT @Klopfe @a-rahimi?